crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/class_info.rs

```diff
@@ -275,6 +279,163 @@ mod get_utility_info_tests {
     }
 }
 
+// variants
+// -------
+
+/// The result of matching a variant against a target.
+#[derive(Debug, Eq, PartialEq)]
+enum VariantMatch {
+    /// The variant matches an exact target.
+    Exact,
+    /// The variant matches a partial target.
+    Partial,
+    /// The variant does not match the target.
+    None,
+}
+
+impl VariantMatch {
+    /// Checks if a variant matches a target, and returns the result.
+    fn from(target: &str, variant_text: &str) -> VariantMatch {
```

Comment:
Nit: you could implement using the `From` trait like this:

```rs
impl From<(&str, &str)> for VariantMatch {
	fn from((target, variant): (&str, &str)) -> Self {

	}
}
```

This is way better, and you can things like `("s-md", "s-md").into()`